sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,275 @@
+        - bearerAuth: []
+  /info/datasets:
+    get:
+      description: Returns a list of the datasets the user has access to, an empty list is a valid response.
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              example: ["aa-dataset-123456-asdfgh", "bb-dataset-123456-asdfgh"]
+              schema:
+                type: array
+        "401":
+          description: Authentication failure
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/datasets/{datasetId}:
+    get:
+      description: Returns an array with metadata about the dataset
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: datasetId
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset of interest
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DatasetInfo"
+          description: successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/datasets/{datasetId}/files:
+    get:
+      description: Returns an array with metadata about all files in a dataset
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: datasetId
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset of interest
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/FileInfo"
+          description: successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /ready:
```

Comment:
Not `/health`? https://github.com/neicnordic/sensitive-data-archive/blob/5218110d6ccc75d8682d42229b0b5d03ef3801b5/sda-download/api/api.go#L74

Reply:
`health` or `ready` serve the same purpose

Reply:
I know, I was just wondering if it would make sense to keep the same endpoint name :upside_down_face:

Reply:
If there's no special reason, I agree with @MalinAhlberg that it is better to keep the same endpoint name.